app/scripts/modules/netflix/pipeline/stage/chap/chapExecutionDetails.html

```diff
@@ -34,8 +34,11 @@
           <div>
             <a href="{{runUrl}}" target="_blank">Run Details</a>
           </div>
-          <div>
-            <a href="{{canaryReportUrl}}" target="_blank">Canary Report</a>
+          <div ng-repeat="canaryReport in canaryReports">
+            <a href="{{canaryReport.url}}" target="_blank">
+              Canary Report
+              <span ng-show="canaryReports.length > 1"> ({{ canaryReport.key }})</span>
```

Comment:
no good reason.  I tend to think of `ng-show` as being more lightweight since it doesn't create additional scope, (although it adds DOM overhead).  You prefer `ng-if`?